tests/hicparam/hicparam.nf.test

```diff
@@ -0,0 +1,118 @@
+nextflow_function {
+
+    name "checkHiCParam"
+    script "./hicparam.nf"
+    function "checkHiCParam"
+
+    test("Dummy_hic.R{1,2}.fq.gz") {
```

Comment:
@SarahBailey1998, @ignacio3437

The permissible patterns are tested here. Are we missing something?

Reply:
would "dummy_hic_{1,2).merged.fastq.gz be allowed?

Reply:
If you mean: `dummy_hic_{1,2}.merged.fastq.gz` and not `dummy_hic_{1,2).merged.fastq.gz`, then yes.

The permissible pattern is `^\S+\{1,2\}[\w\.]*\.f(ast)?q\.gz$`

\S+ will cover the dummy_hic_ part

We can test it here: https://regex101.com

Reply:
yeah that's what I mean, I made a typo sorry! Awesome